eng/pipelines/libraries/stress/http.yml

```diff
@@ -21,12 +21,15 @@ variables:
     value: $(sourcesRoot)/System.Net.Http/tests/StressTests/HttpStress
   - name: sdkBaseImage
     value: dotnet-sdk-libraries-current
-
+  - name: shareRoot
+    value: $(System.DefaultWorkingDirectory)/share
```

Comment:
Why not to place this under `$(Build.ArtifactStagingDirectory)` instead of defining a custom location?

Reply:
Honestly, I didn't know about this pre-defined variable. I could get rid of `shareRoot` then.
Either way, there still have to be different share paths for client and server, since the dumps can have (and do at least on Linux) the exact same file name. Within the docker image, we don't know if we're client or server unless we parse the stress args or pass the info in separately (something I didn't want to embark on in this PR, but might be better thing to do in the grand scheme of both our changes).

Reply:
You can consider placing stuff under different subdirectories, the containers are aware if they are client or server (this is somewhat related to my next comment).

Reply:
Changed to ` $(Build.ArtifactStagingDirectory)`